### ISSUE TITLE
TableRegistry::config() does not work properly

### ISSUE THREAD

**Issue by atkrad** [cakephp/cakephp#6891]
Hello, 
I'm using CakePHP ORM as a library in a separate project

```php
TableRegistry::config('Users', ['className' => 'Custom\UsersTable']);
```
But 
```php 
var_dump(TableRegistry::get('Users'));
```
retern "Cake\ORM\Table" object

**Comment by JD-Robbs** [comment]
I suspect you'll need to reference your namespace - `App` by default. I.e.:

    TableRegistry::config('Users', ['className' => 'App\Custom\UsersTable']);

**Comment by markstory** [comment]
Is your classname autoloadable? This may be fixed by the changes in #6824

**Comment by atkrad** [comment]
Thanks @markstory and @JD-Robbs 
I changed the version from "3.0.7" to "dev-master" and resolved it
